sotodlib/tod_ops/flags.py

```diff
@@ -998,3 +1021,73 @@ def noise_fit_flags(aman, low_wn, high_wn, high_fk):
         return flag_valid_fk
     else:
         return None
+
+def get_noisy_subscan_flags(aman, subscan_stats_T, subscan_stats_Q, subscan_stats_U,
```

Comment:
Making this require T, Q, and U simultaneously makes this a very restrictive function (i.e. can only be run for the SATs not LAT and only after demodulation). Maybe we can make this take just one signal (i.e. signal = subscan_stats_T) and produce one dets x subscan boolean mask/RangesMatrix per signal then the preprocess method could choose how to combine these i.e. if there's one use 1, if there's >1 use logical and or logical or, etc. Thoughts?